PocketKit/Sources/PocketKit/MyList/ReaderActionsWebActivity/ReaderActionsWebActivity.swift

```diff
@@ -0,0 +1,63 @@
+import UIKit
+import Textile
+
+enum WebActivityTitles: String {
+    case save = "Save"
+    case report = "Report"
+    case delete = "Delete"
+    case archive = "Archive"
+    case favorite = "Favorite"
+    case unfavorite = "Unfavorite"
+    case moveToSaves = "Move to saves"
+
+    func getImageAssetIcon() -> ImageAsset {
```

Comment:
```suggestion
    var imageAsset: ImageAsset {
```

This isn't really a "function", per se, more of a computed property. For cleanliness!